styles.css

```diff
@@ -1,9 +1,13 @@
-.container {
+body {
   font-family: "Poppins", sans-serif;
   font-style: normal;
+}
```

Comment:
- [Optional] Kindly, remove the default body `margin`, because it may be difficult to style it, within the default value. so it is better to remove all element default `margin` and `padding` by using `*` selector. 👍
`* {
    margin: 0;
    padding: 0;
}` 😍😒